include/fcio.h

```diff
@@ -5,8 +5,8 @@
     Details.
 */
 
-#ifndef __FCIO
-#define __FCIO
+#ifndef __MEGA65_FCIO_H
+#define __MEGA65_FCIO_H
 
 #include <memory.h>
```

Comment:
Would it be a good idea to put the header files in a subdirectory to make it more clear that the file belongs to mega65?
```
#include <mega65/memory.h>
```
The idea is that it avoids clashes and makes it more clear to which system the header file belongs to. It is especially helpful if source files using the `#include` is shared between multiple systems, e.g. shared with systems other than MEGA65.

Reply:
I like the idea and has created a separate issue. It would break compatibility, so we would need to agree.